### DISCUSSION TITLE
Sidebar does not keep the state after refreshing the page

### DISCUSSION THREAD

**Discussion by goth-pl** [directus/directus#15863] [Feature Requests]
### Describe the Bug

Sidebar does not keep the state after refreshing the page.

Maybe it's not a bug, but could be improved :)

#### For `window.width` >= 1430 (sidebar is open by default)
1. Open the page
2. Close the sidebar
3. Refresh the page

Current behavior: Sidebar is open.
Expected behavior: Sidebar is closed.

#### For `window.width` < 1430 (sidebar is closed by default)
1. Open the page
2. Open the sidebar
3. Refresh the page

Current behavior: Sidebar is closed.
Expected behavior: Sidebar is open.

### What version of Directus are you using?

9.18.1

### What version of Node.js are you using?

16

### What database are you using?

Postgres 14

### What browser are you using?

Chrome

### How are you deploying Directus?

running locally

**Comment by br41nslug** [comment]
Moving this to a discussion to gauge community interest in this improvement 😄

**Comment by br41nslug** [comment]
Linked PR https://github.com/directus/directus/pull/15861

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
